elegant/gui/pose_annotation.py

```diff
@@ -169,6 +180,32 @@ def _update_widget(self, center_tck, width_tck):
         self.outline.geometry = (center_tck, width_tck)
         self._enable_buttons()
 
+    def auto_center(self):
```

Comment:
Lots of duplicated code between `auto_center` and `auto_widths`: prime candidate for refactoring out into a shared private function (with leading `_` in the name).